src/domains/preference/preference.domain.ts

```diff
@@ -0,0 +1,104 @@
+import { AccessTokenDomain } from '../auth/access-token.domain'
+import { IPreference } from './index.interface'
+import {
+  PreferenceDoc,
+  PreferenceModel,
+  PreferenceProps,
+} from '@/schemas/preference.schema'
+import { BadRequestError } from '@/errors/400/index.error'
+import { GlobalLanguageCode } from '@/global.interface'
+import { PutPreferenceDto } from '@/dto/put-preference.dto'
+import { DeleteForbiddenError } from '@/errors/403/action_forbidden_errors/delete-forbidden.error'
+
+export class PreferenceDomain {
+  private readonly props: Partial<IPreference>
+
+  private constructor(props: Partial<IPreference>) {
+    this.props = props
+  }
+
+  get id() {
+    return this.props.id
+  }
```

Comment:
Follow the order of word domain

Reply:
Handled in https://github.com/ajktown/api/pull/58/commits/c22f64102077fa43c0d64f272467283ab5879d51